workers/META.yml

```diff
@@ -0,0 +1,13 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1071345
+      results:
+        - test: dedicated-worker-parse-error-failure.html
+    - product: chrome
+      url: https://crbug.com/906850
+      results:
+        - test: opaque-origin.html
+    - product: chrome
+      url: https://crbug.com/795636
```

Comment:
This is a bug about upstreaming tests, and doesn't say anything about the failure.

Reply:
Reverted